README.md

```diff
@@ -1,26 +1,35 @@
 # GeoUtils
 Set of tools to handle raster and vector data sets in Python.
 
-This package offers Python classes and functions as well as command line tools to work with both geospatial raster and vector datasets. It is built upon rasterio and GeoPandas. In a single command it can import any geo-referenced dataset that is understood by these libraries, complete with all geo-referencing information, various helper functions and interface between vector/raster data.
-
-
-## Installation ##
-
-* Create environment and install dependencies
+![](https://readthedocs.org/projects/geoutils/badge/?version=latest)
+[![build](https://github.com/GlacioHack/GeoUtils/actions/workflows/python-app.yml/badge.svg)](https://github.com/GlacioHack/GeoUtils/actions/workflows/python-app.yml)
+[![Conda Version](https://img.shields.io/conda/vn/conda-forge/geoutils.svg)](https://anaconda.org/conda-forge/geoutils)
+[![Conda Platforms](https://img.shields.io/conda/pn/conda-forge/geoutils.svg)](https://anaconda.org/conda-forge/geoutils)
+[![Conda Downloads](https://img.shields.io/conda/dn/conda-forge/geoutils.svg)](https://anaconda.org/conda-forge/geoutils)
 
-`conda env create -f environment.yml`
+This package offers Python classes and functions as well as command line tools to work with both geospatial raster and vector datasets. It is built upon rasterio and GeoPandas. In a single command it can import any geo-referenced dataset that is understood by these libraries, complete with all geo-referencing information, various helper functions and interface between vector/raster data.
 
-* Activate environment
 
-`conda activate geoutils`
+## Installation
 
-* Install this package
+#### With conda (recommended)
+First, set a strict channel priority. This seems to be optional (but recommended) on UNIX-based systems, but essential for Windows:
+```bash
+conda config --add channels conda-forge
```

Comment:
That is a bit intrusive to ask the users to change their global conda settings. Isn't there a way to set the channel only for this install?

Reply:
Yes, I just learned. Using `conda install --strict-channel-priority`

Reply:
Replaced this in 15fa5df